packages/template-retail-react-app/app/ssr.js

```diff
@@ -67,6 +69,49 @@ const options = {
     encodeNonAsciiHttpHeaders: true
 }
 
+const tenantIdRegExp = /^[a-zA-Z]{4}_([0-9]{3}|s[0-9]{2}|stg|dev|prd)$/
+const shortCodeRegExp = /^[a-zA-Z0-9-]+$/
+
+/**
+ *  Handles JWKS (JSON Web Key Set) caching the JWKS response for 2 weeks.
+ *
+ * @param {object} req Express request object.
+ * @param {object} res Express response object.
+ * @param {object} options Options for fetching B2C Commerce API JWKS.
+ * @param {string} options.shortCode - The Short Code assigned to the realm.
+ * @param {string} options.tenantId - The Tenant ID for the ECOM instance.
+ * @returns {Promise<*>} Promise with the JWKS data.
+ */
+async function jwksCaching(req, res, options) {
```

Comment:
nit: would we consider moving this method, tenantIdRegExp, and shortCodeRegExp into `jwt-utils` to reduce the code in ssr.js?